webapp/src/main/java/com/box/l10n/mojito/service/scheduledjob/ScheduledJobStatusRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.box.l10n.mojito.service.scheduledjob;
+
+import com.box.l10n.mojito.entity.ScheduledJobStatusEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface ScheduledJobStatusRepository
+    extends JpaRepository<ScheduledJobStatusEntity, Long> {
+  @Query("SELECT sjs FROM ScheduledJobStatusEntity sjs " + "WHERE sjs.jobStatus = :jobStatus")
+  ScheduledJobStatusEntity findByEnum(ScheduledJobStatus jobStatus);
```

Comment:
why do we need the plus here for concatentation?

Reply:
Maven spotless does this.